### PR TITLE
feature/SM-6107-Remove date validation for adding reinstatements

### PR DESCRIPTION
**Pull request by RichardEustace**

### CODE REVIEW

**Comment by RichardEustace** [comment]
Business rules updated separately as per -https://kainossoftwareltd.sharepoint.com/:w:/r/sites/StreetManager_External/_layouts/15/Doc.a[…]direct=true&cid=e7922f79-6eca-4fb6-8337-0f50cb8d728b

File: docs/api-documentation/V1.25.11/changelog.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ Updated Work API with the following changes:
   <li>
     SM-6795: Added new validation rules to Work API that prevent users belonging to suspended organisations from logging in and to prevent contractor orgs from performing any read/write actions on suspended organisations records
   </li>
+  <li>
+    SM-6107: Updated the validation on <code>POST works/{workReferenceNumber}/sites</code> and <code>POST works/{workReferenceNumber}/sites/{siteNumber}/reinstatements</code> to use the earliest actual_start_date across all permits associated with a work.
```
Comment (caoimheb) [inline docs/api-documentation/V1.25.11/changelog.md:24]:
```suggestion
    SM-6107: Updated the validation on <code>POST works/{workReferenceNumber}/sites</code> and <code>POST works/{workReferenceNumber}/sites/{siteNumber}/reinstatements</code> to use the earliest actual_start_date across all permits associated with a work when validating reinstatement_date. The reinstatement date cannot be before the earliest actual_start_date.
```
? Or some detail on what the actual_start_date is used for?

Comment (pwright08) [inline docs/api-documentation/V1.25.11/changelog.md:24]:
```suggestion
    SM-6107: Updated the validation on <code>POST /works/{workReferenceNumber}/sites</code> and <code>POST /works/{workReferenceNumber}/sites/{siteNumber}/reinstatements</code> to use the earliest <code>actual_start_date</code> across all permits associated with a work.
```

File: docs/api-documentation/V2.10.6/changelog.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ Updated Work API with the following changes:
   <li>
     SM-6795: Added new validation rules to Work API that prevent users belonging to suspended organisations from logging in and to prevent contractor orgs from performing any read/write actions on suspended organisations records
   </li>
+  <li>
+    SM-6107: Updated the validation on <code>POST works/{workReferenceNumber}/sites</code> and <code>POST works/{workReferenceNumber}/sites/{siteNumber}/reinstatements</code> to use the earliest actual_start_date across all permits associated with a work.
```
Comment (pwright08) [inline docs/api-documentation/V2.10.6/changelog.md:24]:
```suggestion
    SM-6107: Updated the validation on <code>POST /works/{workReferenceNumber}/sites</code> and <code>POST /works/{workReferenceNumber}/sites/{siteNumber}/reinstatements</code> to use the earliest <code>actual_start_date</code> across all permits associated with a work.
```

File: docs/api-documentation/V1.25.11/changelog.md [hunk 3]
```diff
@@ -20,6 +20,9 @@ Updated Work API with the following changes:
   <li>
     SM-6795: Added new validation rules to Work API that prevent users belonging to suspended organisations from logging in and to prevent contractor orgs from performing any read/write actions on suspended organisations records
   </li>
+  <li>
+    SM-6107: Updated the validation on <code>POST /works/{workReferenceNumber}/sites</code> and <code>POST /works/{workReferenceNumber}/sites/{siteNumber}/reinstatements</code> to use the earliest <code>actual_start_date</code> across all permits associated with a work when validating reinstatement_date. The reinstatement date cannot be before the earliest <code>actual_start_date</code>.
```
Comment (pwright08) [inline docs/api-documentation/V1.25.11/changelog.md:24]:
```suggestion
    SM-6107: Updated the validation on <code>POST /works/{workReferenceNumber}/sites</code> and <code>POST /works/{workReferenceNumber}/sites/{siteNumber}/reinstatements</code> to use the earliest <code>actual_start_date</code> across all permits associated with a work when validating <code>reinstatement_date</code>. The reinstatement date cannot be before the earliest <code>actual_start_date</code>.
```

File: docs/api-documentation/V2.10.6/changelog.md [hunk 4]
```diff
@@ -20,6 +20,9 @@ Updated Work API with the following changes:
   <li>
     SM-6795: Added new validation rules to Work API that prevent users belonging to suspended organisations from logging in and to prevent contractor orgs from performing any read/write actions on suspended organisations records
   </li>
+  <li>
+    SM-6107: Updated the validation on <code>POST /works/{workReferenceNumber}/sites</code> and <code>POST /works/{workReferenceNumber}/sites/{siteNumber}/reinstatements</code> to use the earliest <code>actual_start_date</code> across all permits associated with a work when validating reinstatement_date. The reinstatement date cannot be before the earliest <code>actual_start_date</code>.
```
Comment (pwright08) [inline docs/api-documentation/V2.10.6/changelog.md:24]:
```suggestion
    SM-6107: Updated the validation on <code>POST /works/{workReferenceNumber}/sites</code> and <code>POST /works/{workReferenceNumber}/sites/{siteNumber}/reinstatements</code> to use the earliest <code>actual_start_date</code> across all permits associated with a work when validating <code>reinstatement_date</code>. The reinstatement date cannot be before the earliest <code>actual_start_date</code>.
```

**Review by pwright08 — APPROVED** [review]

**Review by caoimheb — APPROVED** [review]

**Pull request merged**